keyboards/handwired/symmetric70_proto/keymaps/default/keymap.c

```diff
@@ -0,0 +1,57 @@
+/* Copyright 2020 mtei
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include QMK_KEYBOARD_H
+
+// Defines names for use in layer keycodes and the keymap
+enum layer_names {
+    _BASE,
+    _FN
+};
+
+#define KC_FUNC MO(_FN)
+
+// clang-format off
+
+#define KC___ KC_TRNS
+#define KC_xx KC_NO
```

Comment:
2. I'm personally not a fan of the use of these sorts of keycode aliases, and would much prefer the QMK-native `_______` and `XXXXXXX` be used.

Reply:
I don't like the keymap descriptions to be long in the horizontal direction.
However, I compromised a bit and changed it to use `KC_NO` and` ________`.